### PR TITLE
Add swagger package documentation

### PR DESCRIPTION
**Pull request by praminda**

### CODE REVIEW

File: stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal [hunk 1]
```diff
@@ -35,58 +39,76 @@ public type ServiceInformation {
...
     DocumentationInformation externalDocs,
 };
 
-@Description {value: "Model for security requirement definition. This is most likely the oauth scopes"}
-@Field {value: "name: Security scheme name"}
-@Field {value: "requirements: Array of security requirements"}
+documentation {
+    Model for security requirement definition. This is most likely the oauth scopes.
+
+	F{{name}} Security scheme name
```
Comment (Shan1024) [inline stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal]: Seems like there are some inconsistancies when documenting fields, etc. Some are aligned with with the description and some are not. Shall we align with description in all places?

Comment (praminda) [reply stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal]: @Shan1024 Checked locally. Code is properly formatted. Should be some issue in github diff. I've seen this happening before

Comment (praminda) [reply stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal]: Replaced tabs with spaces. Please check now

File: stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal [hunk 2]
```diff
@@ -35,58 +39,76 @@ public type ServiceInformation {
...
-@Field {value: "name: parameter name"}
-@Field {value: "description: Description of the parameter"}
-@Field {value: "required: Is this paramter MUST be present in the request"}
-@Field {value: "discontinued: Is this parameter deprecated"}
-@Field {value: "allowEmptyValue: is empty values allowed for this parameter. Valid only for query parameters"}
-@Field {value: "schema: Parameter data type"}
+documentation {
+    Model for keeping swagger parameter information.
+
+	F{{inInfo}} Where the parameter is located. Ex: query
```
Comment (shafreenAnfar) [inline stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal]: Indentation is wrong..

File: stdlib/ballerina-http/src/main/ballerina/swagger/Package.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+This client can be used in client applications to call the service defined in the Swagger file.
+
+### Service to Swagger Export
+`ballerina swagger export <balfile> 
+    [-o <path>|--output<path>]
+    [-s <servicename>|--service <servicename>]`
+
+Export the Ballerina service to a definition of OpenApi Specification 3.0.
+For the export to work properly, the input ballerina service should be defined using basic service and resource level HTTP annotations.
```
Comment (appellj) [inline stdlib/ballerina-http/src/main/ballerina/swagger/Package.md]: Change "ballerina" to "Ballerina"

File: stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal [hunk 4]
```diff
@@ -15,14 +15,18 @@
...
-@Description {value: "Model for additional swagger information of a ballerina service"}
-@Field {value: "title: Title of the swagger definition"}
-@Field {value: "serviceVersion: Version of the swagger API"}
-@Field {value: "termsOfService: Service usage terms and conditions"}
-@Field {value: "contact: Contact information for the exposed API."}
-@Field {value: "license: License information for the exposed API."}
-@Field {value: "externalDoc: Additional external documentation."}
-@Field {value: "tags: A list of tags used by the specification with additional metadata"}
+documentation {
+    Model for additional swagger information of a ballerina service.
```
Comment (appellj) [inline stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal]: Change "swagger" to "Swagger" and "ballerina" to "Ballerina" in descriptions throughout the documentation sections in this file

File: stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal [hunk 5]
```diff
@@ -35,58 +39,76 @@ public type ServiceInformation {
...
-@Field {value: "schema: Parameter data type"}
+documentation {
+    Model for keeping swagger parameter information.
+
+    F{{inInfo}} Where the parameter is located. Ex: query
+    F{{name}} Parameter name
+    F{{description}} Description of the parameter
+    F{{required}} Is this parameter MUST be present in the request
+    F{{discontinued}} Is this parameter deprecated
+    F{{allowEmptyValue}} Is empty values allowed for this parameter. Valid only for query parameters
```
Comment (appellj) [inline stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal]: Change "Is empty values" to "Is an empty value"

File: stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal [hunk 6]
```diff
@@ -35,58 +39,76 @@ public type ServiceInformation {
...
     string name,
     string description,
     DocumentationInformation externalDocs,
 };
 
-@Description {value: "Model for security requirement definition. This is most likely the oauth scopes"}
-@Field {value: "name: Security scheme name"}
-@Field {value: "requirements: Array of security requirements"}
+documentation {
+    Model for security requirement definition. This is most likely the oauth scopes.
```
Comment (appellj) [inline stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal]: oauth -> OAuth

File: stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal [hunk 7]
```diff
@@ -119,6 +149,15 @@ public type ResourceInformation {
...
 };
 
+documentation {
+    Model for keeping swagger response information.
+
+    F{{code}} Reponse code
+    F{{description}} Response description
+    F{{response}} Response content
+    F{{headers}} Response headers
+    F{{examples}} Examples for this reponse
```
Comment (appellj) [inline stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal]: reponse -> response

File: stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal [hunk 8]
```diff
@@ -127,24 +166,44 @@ public type Response {
...
 public type Header {
-    string name,
+    boolean required,
+    boolean discontinued,
     string description,
-    string headerType,
 };
 
+documentation {
+    Model for keeping swagger Example information.
```
Comment (appellj) [inline stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal]: Example -> example

File: stdlib/ballerina-http/src/main/ballerina/swagger/Package.md (hunk 3 above)
Comment (praminda) [reply stdlib/ballerina-http/src/main/ballerina/swagger/Package.md]: @appellj Fixed all comments with 66bff6aeb27c4659eaaa6f9c192e47eccb697f75

**Pull request merged**